src/main/views/admin-dashboard.njk

```diff
@@ -13,6 +14,25 @@
     <div class="govuk-grid-row parent-box">
         <h1 class="govuk-heading-l">{{ heading }}</h1>
     </div>
+    {% set html %}
```

Comment:
A small issue. It might be better to move the "set" to after line 31 to this does not need to be set for other user roles as it is unnecessary.